test-support/helpers/a11y/helpers/actions.js

```diff
@@ -14,7 +14,8 @@ const FOCUS_SELECTORS = [
   'a[href]',
   'area[href]',
   'iframe',
-  '[tabindex]:not([tabindex="-1"])'
+  '[tabindex]:not([tabindex="-1"])',
+  'form'
```

Comment:
I think introducing another array to check against would be the best approach.

Maybe something like `FOCUS_EXCEPTIONS`. Then the logic can be make sure it is focusable OR that it is one of the exceptions to that rule.